lib/Basics/ReadWriteLock.h

```diff
@@ -104,7 +104,7 @@ class ReadWriteLock {
 
   static constexpr uint64_t WRITE_LOCK = 1;
 
-  static constexpr uint64_t READER_INC = 1ul << 32;
+  static constexpr uint64_t READER_INC = 1ull << 32;
```

Comment:
This change should be removed now, because it will conflict with the most recent upstream change by @mpoeter in the same file.

Reply:
I will merge in `devel` and then things should be good again.

Reply:
Done.